### DISCUSSION TITLE
v2.4.2

### DISCUSSION THREAD

**Discussion by ehsandeep** [projectdiscovery/nuclei#887] [Releases]
<hr /><em>This discussion was created from the release <a href='https://github.com/projectdiscovery/nuclei/releases/tag/v2.4.2'>v2.4.2</a>.</em>

**Comment by ehsandeep** [comment]
This version includes a hotfix for a flaw in Nuclei [v2.4.1](https://github.com/projectdiscovery/nuclei/tree/v2.4.1) that prevented it from parsing input with capital characters; the issue was reported by @pdelteil and explained here - #874, and it only affects users using Nuclei on **Linux**.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
